DXRMapFixups/DeusEx/Classes/DXRFixupM15.uc

```diff
@@ -53,202 +53,263 @@ function CheckConfig()
     Super.CheckConfig();
 }
 
-function PreFirstEntryMapFixes()
+function PreFirstEntryMapFixes_Bunker()
```

Comment:
this diff is hard to read, here's the changes for it https://github.com/Die4Ever/deus-ex-randomizer/commits/fcf6b6d1427570c13c04e94acc9685e6f18d1008/DXRMapFixups/DeusEx/Classes/DXRFixupM15.uc

- Fully prevent Tong and Everett ending skips for real (fixes #408)
- Duplicate some Joe Greene newspaper articles so that the bingo goal is easier (work towards #461)
- Refactor PreFirstEntryMapFixes for mission 15 fixups so that the ifdef doesn't have a huge number of lines in it
- Create dupes of all the clone datacubes in Area 51 page to simplify the bingo goal (work towards #461)
- Add four new clone datacubes to Page. Work towards #461
- RATS RATS RATS RATS RATS RATS (fixes #494)
- Make generic datacube spawning functions and modify existing spawns to use the appropriate ones